plinth/views.py

```diff
@@ -51,6 +54,31 @@ def index(request):
                              'details_label': details_label})
 
 
+def uninstall(request, modulename):
```

Comment:
I believe this should be handled by the SetupView and SetupMiddleware.  The transaction handling part to should go to SetupHelper.